package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "xhr",
+  "name": "@videojs/xhr",
```

Comment:
I grabbed whatever is currently in `main`, I think it's showing up because this branch is based on a different commit.